.github/run_translation_verification.sh

```diff
@@ -16,14 +16,20 @@ fi
 
 # Verify that translations are present for all languages
 cd conf/i18n/translations
-if [[ $CIRCLE_BRANCH == release/*
-  || $CIRCLE_BRANCH == hotfix/*
-  || $CIRCLE_BRANCH == master
-  || $CIRCLE_BRANCH == support/* ]]
+
+exit_code=0
+if [[ $GITHUB_REF_NAME == release/*
+  || $GITHUB_REF_NAME == hotfix/*
+  || $GITHUB_REF_NAME == master
+  || $GITHUB_REF_NAME == support/* ]]
 then
   for po_file in *.po
-    do msgcmp $po_file messages.pot
+  do
+    msgcmp $po_file messages.pot
+    exit_code=$(( exit_code + $? ))
```

Comment:
Can this be `msgcmp $po_file messages.pot || exit_code=1` instead incrementing the exit code?

Reply:
yes, updated! I thought it might be nice to have the exit code reflect how many po files had errors, but that's probably not standard exit code behavior (though from reading online it sounds like exit codes in general are a little fuzzy)